### PR TITLE
Replace wwrelay-utils with pe-utils

### PR DESCRIPTION
**Pull request by mray190**
Move in config and template files into snap-pelion-edge from wwrelay-utils
Remove wwrelay-utils part from the snap
Add in pe-utils part for identity.json generation

Note: Configuration files and templates have been moved around in these commits
DeviceDB is disabled to start, with a commit that we can revert later to turn back on

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -325,123 +325,17 @@ parts:
...
-      stage:
-        - $etc
-        - $conf
-        - $versions
-        - $utils
-        - $debug
-        - $devtools
-        - $system
-        - $support
+        install ${SNAPCRAFT_PROJECT_DIR}/files/wwrelay-utils/version.json ${SNAPCRAFT_PART_INSTALL}/wigwag/etc/
```
Comment (costanic) [inline snap/snapcraft.yaml:335]: since the part was renamed to pe-utils, we should move these files to a pe-utils folder.

Comment (mray190) [reply snap/snapcraft.yaml:335]: Good catch - will fix that right now

**Comment by mray190** [comment]
Wait for v1.18.1 tag before merging to `dev`

**Comment by mray190** [comment]
Note that PR #276 was incorrectly merged. I rebased and removed that commit from this PR, and opened a new PR that we will keep closed for now. The icon says `merged` in this PR - that is NOT the case. Ignore it

**Review by costanic — APPROVED** [review]

**Pull request merged**